package/skelcd-control-SLES.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Oct 11 11:24:17 UTC 2016 - mfilka@suse.com
+
+- Removed reference to netprobe client. It was used in AutoYaST
+  only and everything what this client does should be currently
+  handled by wicked.
+- 12.2.3 
```

Comment:
Hmm, this is not going to SP2, is it? Then we should use 12.3.0 to mark this as SP3.